collect_app/src/main/java/org/odk/collect/android/activities/FormDownloadListActivity.java

```diff
@@ -685,7 +686,7 @@ public static String getDownloadResultMessage(Map<ServerFormDetails, String> res
         for (ServerFormDetails k : keys) {
             b.append(k.getFormName() + " ("
                     + ((k.getFormVersion() != null)
-                    ? (TranslationHandler.getString(Collect.getInstance(), R.string.version) + ": " + k.getFormVersion() + " ")
+                    ? (getLocalizedString(Collect.getInstance(), R.string.version) + ": " + k.getFormVersion() + " ")
```

Comment:
This whole method seems to be unused so please remove it at all.

Reply:
Done.